sda/internal/database/db_functions.go

```diff
@@ -696,3 +691,30 @@ func (dbs *SDAdb) getCorrID(user, path string) (string, error) {
 
 	return corrID, nil
 }
+
+func (dbs *SDAdb) ListActiveUsers() ([]string, error) {
```

Comment:
```suggestion
// ListActiveUsers returns a list with all users who have submitted files that are not mapped to a dataset
func (dbs *SDAdb) ListActiveUsers() ([]string, error) {
```

It would be nice to add a comment description for `getCorrID` and other functions if they're missing it.